anime/assets/script.js

```diff
@@ -0,0 +1,22 @@
+let menu = document.querySelector('#menu-icon');
+let navbar = document.querySelector('.navbar');
+
+menu.onclick = () => {
+    menu.classList.toggle('bx-x');
```

Comment:
As much as possible we are using 2 space tabs for our indentation. Kindly revise, thanks

Reply:
Okay po Sir. Thank you so much for the feedback!